src/Cms/Plugin.php

```diff
@@ -212,6 +214,53 @@ public function toArray(): array
 		];
 	}
 
+	/**
+	 * Returns the update status object unless the
+	 * update check has been disabled for the plugin
+	 *
+	 * @param array|null $data Custom override for the getkirby.com update data
+	 */
+	public function updateStatus(array|null $data = null): UpdateStatus|null
```

Comment:
Wondering if `updates()` would be a more Kirby-like name and still fitting.

Reply:
Thought about it as well. I find `updates()` a bit weird because there can only be a single update. But a singular `update()` sounds like a verb. So I decided on `updateStatus()` even though it's also not perfect.

Reply:
I was thinking again about it, but is it really a single update when I'm on version 3.7.1 and the system gives me info on updating to 3.7.5 - or is that four updates?

Reply:
The class only ever suggests a single update. It is the user's responsibility to decide whether intermediate steps are useful and/or necessary.

Reply:
It suggests a version number, right? My point was that this would/could suggest multiple updates - which would make the name not that strange at all.

Reply:
Suggesting multiple updates would require a rewrite because there can only be one return value for each of the methods. Also I feel like multiple updates would be too overwhelming (e.g. if there is a security update, the user shouldn't need to care that there is also a feature update and an upgrade available – until that security update is done).

Reply:
Again, I'm not talking about changing anything but the name. Returning an info about 3.7.5 when you're on 3.7.1 is info about updates, not just one. And since I think the current name is a bit cumbersome, my only argument here is to use `Updates` and that it does make sense. Not to change anything how it works or what is returned.

Reply:
But if you like the current name that much, fine, we can stick with it.

Reply:
I don't like either name 100 %. :)

Reply:
Let's stay with updateStatus for now.